pkg/printers/externalversion/BUILD

```diff
@@ -0,0 +1,90 @@
+load("@io_bazel_rules_go//go:def.bzl", "go_library", "go_test")
+
+go_library(
+    name = "go_default_library",
+    srcs = ["describe.go"],
+    importpath = "k8s.io/kubernetes/pkg/printers/externalversion",
+    visibility = ["//visibility:public"],
+    deps = [
+        "//pkg/api/v1/events:go_default_library",
```

Comment:
These thirteen dependencies are all problematic if this is moved into `pkg/kubectl`. We are currently at twenty-two remaining internal dependencies before `kubectl` independence. Moving this file into `pkg/kubectl` will dramatically increase this number.